README.md

```diff
@@ -89,6 +90,17 @@ For Windows users, the commands are slightly different:
 
 Screenshots of the results will be available in the `reports/` folder.
 
+## Troubleshooting
+Chrome drivers might crash due to the small size of `/dev/shm` in the docker container:
```

Comment:
Can't this be simply solved by adding `--shm-size=1g` to the run command?

Reply:
Never used this flag before but you are probably right. I will try it asap.